<!doctype html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en"> <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">

  <title>BioCore: Mak Snp Analysis</title>
  <meta name="description" content="">
  <meta name="author" content="">

  <meta name="viewport" content="width=device-width,initial-scale=1">
  
  <!-- <link href='http://fonts.googleapis.com/css?family=Josefin+Slab' rel='stylesheet' type='text/css'> -->
  <!-- <link rel="stylesheet" href="js/libs/coin/coin-slider-styles.css" type="text/css" /> -->
  <link rel="stylesheet" href="css/style.css">

  <script src="js/libs/modernizr-2.0.6.min.js"></script>
</head>

<body>
  <div id="container">
    <header>
      <nav>
        <ul>
          <li><a href="/">&larr; home</a></li>
        </ul>
        </nav>
    </header>
    <div class="main" role="main">
      <div id="content">
      <h1 id="title">Mak Snp Analysis</h1>
<ul id="toc"></ul>
<h2>Objective</h2>
<div class="section">
<p>To identify SNPs and indels in wild type and mutant C. elegans samples in order to identify the genotypic modification responsible for transgene de-silencing in hjIs21 mutant known as huy116. Wild Type control sample hjIs21 was used as a reference.</p>
</div>
<h2>Data</h2>
<div class="section">
<p>Paired-end data was supplied for both wild type and mutant in the paths below:</p>
<table class="sorted">
	<tr>
		<th>Path </th>
		<th>Sample </th>
	</tr>
	<tr>
		<td> /n/analysis/Mak/hym/Mak-2010-12-10/429GDAAXX/s_1_1_sequence.txt.gz </td>
		<td> hjls21_1 Read 1 </td>
	</tr>
	<tr>
		<td> /n/analysis/Mak/hym/Mak-2010-12-10/429GDAAXX/s_1_2_sequence.txt.gz </td>
		<td> hjls21_1 Read 2 </td>
	</tr>
	<tr>
		<td> /n/analysis/Mak/hym/Mak-2010-12-10/429GDAAXX/s_2_1_sequence.txt.gz </td>
		<td> hjls21_2 Read 1 </td>
	</tr>
	<tr>
		<td> /n/analysis/Mak/hym/Mak-2010-12-10/429GDAAXX/s_2_2_sequence.txt.gz </td>
		<td> hjls21_2 Read 2 </td>
	</tr>
	<tr>
		<td> /n/analysis/Mak/hym/Mak-2010-12-10/429GDAAXX/s_3_1_sequence.txt.gz </td>
		<td> hjls21_3 Read 1 </td>
	</tr>
	<tr>
		<td> /n/analysis/Mak/hym/Mak-2010-12-10/429GDAAXX/s_3_2_sequence.txt.gz </td>
		<td> hjls21_3 Read 2 </td>
	</tr>
	<tr>
		<td> /n/analysis/Mak/hym/Mak-2010-12-10/429GDAAXX/s_4_1_sequence.txt.gz </td>
		<td> hjls21_4 Read 1 </td>
	</tr>
	<tr>
		<td> /n/analysis/Mak/hym/Mak-2010-12-10/429GDAAXX/s_4_2_sequence.txt.gz </td>
		<td> hjls21_4 Read 2 </td>
	</tr>
</table>
<p>Trimmed huy116 reads were used, in the same fashion as the original analysis, for data quality issues as described below.</p>
</div>
<h2>Data Quality</h2>
<div class="section">
<p>Data quality of the raw sequence data was analyzed to ensure downstream analysis is valid.</p>
<p>Fastqc was used for a more in-depth look at the quality of the sequence data. For the hjIs21 sequences, the quality of the read data was satisfactory.  A graph of the read quality for one of the sample files generated by fastqc is provided below:</p>
<p><img src="/img/mak_snp/per_base_quality.png" class="medium" alt="" /></p>
<p>Reads for huy116 samples were found to be of low quality after the 50th base pair. For this reason, these reads were trimmed to 50bp prior to further analysis. A graph of read quality for one of the huy116 samples is shown below:</p>
<p><img src="/img/mak_snp/bad_reads.png" class="medium" alt="" /></p>
<p>The same trimmed sequence files were used from the original analysis.</p>
<p>Statistics were also generated for the quality of the alignment process. Alignment was performed using bwa, with the separate hjIs21 alignments combined into hjIs21_all for variant analysis (described below).</p>
</div>
<h2>Methods</h2>
<div class="section">
<p>Sequences were first aligned using bwa (v0.5.9-r16), aligning each lane separately and then combining the paired-end reads using the bwa sampe tool. Reads were aligned using the ce ws210 reference genome from Ensembl, downloaded from Illumina’s igenome ftp server. The output of the alignment process was a bam file for each set of paired-end reads. Duplicate reads that could be caused by amplification artifacts were removed using samtools rmdup. Then, the multiple hjIs21 bam files were then combined using samtools merge. After which, the picard (v1.49) toolset was utilized to prepare the bam files for <span class="caps">SNP</span> calling. Specifically, picard’s AddOrReplaceReadGroups utility was used to normalize the read groups contained in the bam files to a single group. Also, picard’s ReorderSam tool was used to ensure the bam reads were in the correct order for processing with <span class="caps">GATK</span>.</p>
<p>The <span class="caps">GATK</span> Unified Genotyper (v1.1.10) was used for <span class="caps">SNP</span> and Indel calling. Local realignment was performed initially using <span class="caps">GATK</span> as recommended by their best practices for variant detection. <span class="caps">SNP</span> and indel calling was performed on the locally realigned bam file for both hjIs21 and huy116 samples. Filtering was performed to remove variants with low coverage. The SNPs and indels were then annotated using snpEff(v1.9.5). Custom scripts were then used to mange and group the annotated variants.</p>
</div>
<h2>Analysis</h2>
<div class="section">
<h3><span class="caps">IGV</span> Sessions</h3>
<p><span class="caps">IGV</span> Session:<br />
An <span class="caps">IGV</span> session is available at the link below, which provides visualization of the bam files, snp calls, and indel for huy116. Session was created using <span class="caps">IGV</span> 2.0.5.</p>
<p>igv_session.xml</p>
<p>Full Windows path:<br />
S:\Bioinformatics\analysis\Mak\Mak-2010-01-13_hym\reanalysis\igv_session.xml</p>
<p>Full Mac path:<br />
/Volumes/core/Bioinformatics/analysis/Mak/Mak-2010-01-13_hym/reanalysis/igv_session.xml</p>
<p><strong><span class="caps">NOTE</span>:</strong></p>
<p>The reference genome used for this analysis is not built into the default set of <span class="caps">IGV</span> genomes as of version 2.0.5. For this reason, it will be necessary to add the ws210 c. elegans genome to your <span class="caps">IGV</span> browser. Steps for doing this are provided below</p>
<ul>
	<li>With <span class="caps">IGV</span> open, select File &#8594; Load Genome</li>
	<li>Navigate to the custom genome path:
	<ul>
		<li>S:\Bioinformatics\analysis\Mak\Mak-2010-01-13_hym\reanalysis\ce_ws210.genome</li>
	</ul></li>
	<li>Select open with the ce_ws210.genome file selected. This should add it to your <span class="caps">IGV</span> browser.</li>
</ul>
</div>
<h2><span class="caps">SNP</span> &amp; Indel Analysis</h2>
<div class="section">
<p>The new variant pipeline calls SNPs and indels on the wild type and mutant reads separately. Post-processing  is then used to extract variants only found in the mutant sample. The statistics below provide information for all wild type variants, all mutant variants, variants unique to just mutant – excluding those also found in wild type, and variants unique to just the wild type sample.</p>
<table>
	<tr>
		<td> Sample </td>
		<td> hjIs21 all </td>
		<td> hjIs21 unique </td>
		<td> huy116 all </td>
		<td>  huy116 unique </td>
	</tr>
	<tr>
		<td> Total SNPs </td>
		<td> 1238 </td>
		<td> 233 </td>
		<td> 175 </td>
		<td> 599 </td>
	</tr>
	<tr>
		<td> Intergenic </td>
		<td> 570 </td>
		<td> 127 </td>
		<td> 792 </td>
		<td> 274 </td>
	</tr>
	<tr>
		<td> Intronic </td>
		<td> 607 </td>
		<td> 83 </td>
		<td> 1044 </td>
		<td> 359 </td>
	</tr>
	<tr>
		<td> Synonymous </td>
		<td> 164 </td>
		<td> 27 </td>
		<td> 173 </td>
		<td> 61 </td>
	</tr>
	<tr>
		<td> Non-Synonymous </td>
		<td> 321 </td>
		<td> 69 </td>
		<td> 330 </td>
		<td> 106 </td>
	</tr>
	<tr>
		<td> Stop Gained </td>
		<td> 4 </td>
		<td> 1 </td>
		<td> 6 </td>
		<td> 3 </td>
	</tr>
	<tr>
		<td> Stop Lost </td>
		<td> 2 </td>
		<td> 0 </td>
		<td> 4 </td>
		<td> 2 </td>
	</tr>
	<tr>
		<td> Start Lost </td>
		<td> 0 </td>
		<td> 0 </td>
		<td> 7 </td>
		<td> 6 </td>
	</tr>
</table>
<h3>huy116 unique</h3>
<p>Coverage Histograms provide a sense of the amount of data used to call the individual variants. The histograms below also display the filtering used as part of the variant calling pipeline. Specifically, variants called by <span class="caps">GATK</span> with 10 or fewer reads were excluded from further processing.</p>
<h4>Datasets</h4>
<p>Access to these datasets in Excel format are linked below. Links to locations in the <span class="caps">IGV</span> viewer have been provided for each variant in the excel file. To use, you need to have the <span class="caps">IGV</span> viewer running and the session for this analysis open.</p>
<p>huy116 unique snps:</p>
<p>huy116.unique.snps.info.xls</p>
<p>Full Windows path:<br />
S:\Bioinformatics\analysis\Mak\Mak-2010-01-13_hym\reanalysis\huy116.unique.snps.info.xls</p>
<p>Full Mac path:<br />
/Volumes/core/Bioinformatics/analysis/Mak/Mak-2010-01-13_hym/reanalysis/huy116.unique.snps.info.xls</p>
<p>huy116 unique indels:</p>
<p>huy116.unique.indels.info.xls</p>
<p>Full Windows path:<br />
S:\Bioinformatics\analysis\Mak\Mak-2010-01-13_hym\reanalysis\huy116.unique.indels.info.xls</p>
<p>Full Mac path:<br />
/Volumes/core/Bioinformatics/analysis/Mak/Mak-2010-01-13_hym/reanalysis/huy116.unique.indels.info.xls</p>
<p>hjIs21 unique</p>
</div>

      </div>
    </div>
    <footer>
      <p><a href="/">home</a> | <a href="about.html">about</a> | <a href="contact.html">contact</a></p> 
    </footer>
  </div> <!--! end of #container -->

  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.6.2/jquery.min.js"></script>
  <script>window.jQuery || document.write('<script src="js/libs/jquery-1.6.2.min.js"><\/script>')</script>


  <!-- scripts concatenated and minified via ant build script-->
  <script src="js/libs/jquery.tableofcontents.min.js"></script>
  <script src="js/libs/jquery.tablesorter.min.js"></script>
  <script defer src="js/plugins.js"></script>
  <script defer src="js/script.js"></script>
  <!-- end scripts-->


  <!--[if lt IE 7 ]>
    <script src="//ajax.googleapis.com/ajax/libs/chrome-frame/1.0.3/CFInstall.min.js"></script>
    <script>window.attachEvent('onload',function(){CFInstall.check({mode:'overlay'})})</script>
  <![endif]-->
  
</body>
</html>
